storybook/storybook-css/src/components/Heading/Heading.stories.tsx

```diff
@@ -0,0 +1,144 @@
+/* @license CC0-1.0 */
+
+import { Heading } from '@amsterdam/design-system-react/src'
+import type { Meta, StoryObj } from '@storybook/react'
+import '@amsterdam/design-system-css/src/heading/heading.scss'
+
+const meta = {
+  title: 'CSS Components/Heading',
+  id: 'css-heading',
+  component: Heading,
+  argTypes: {
+    level: {
+      table: {
+        disable: true,
+      },
+    },
+    styleLevel: {
+      options: [1, 2, 3, 4, 5, 6],
+      control: 'select',
+      table: {
+        type: { summary: 'number' },
+        defaultValue: { summary: '1' },
+      },
+    },
+  },
+  args: {
+    children: 'Jouw typograaf biedt mij zulke exquise schreven!',
+    level: 1,
+    styleLevel: 1,
+  },
+  tags: ['autodocs'],
+} as Meta<typeof Heading>
+
+export default meta
+
+/**
+ * Primary Heading used for page titles
+ */
```

Comment:
These descriptions don't really add anything imo

Reply:
This is meant to illustrate to Vincent and Hee Chan the usage of jsdocs. I noticed Hee Chan describing this usage in the components.

Reply:
Removed